src/CommunityPatch/Patches/EquipDraggedNonCivilianItemPatch.cs

```diff
@@ -0,0 +1,296 @@
+      => CommunityPatchSubModule.Error
+        ($"{nameof(EquipDraggedNonCivilianItemPatch)}: Couldn't find {typeName} by reflection.");
+    
+    public override bool? IsApplicable(Game game) {
+      if (UpdateCurrentCharacterIfPossibleMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(UpdateCurrentCharacterIfPossibleMethodInfo));
+        return false;
+      }
+      
+      if (InitializeInventoryMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(InitializeInventoryMethodInfo));
+        return false;
+      }
+      
+      if (AfterTransferMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(AfterTransferMethodInfo));
+        return false;
+      }
+      
+      if (SetIsCivilianItemMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(SetIsCivilianItemMethodInfo));
+        return false;
+      }
+
+      if (RightItemListVmFieldInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(RightItemListVmFieldInfo));
+        return false;
+      }
+      
+      if (LeftItemListVmFieldInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(LeftItemListVmFieldInfo));
+        return false;
+      }
+
+      if (CurrentCharacterFieldInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(CurrentCharacterFieldInfo));
+        return false;
+      }
+      
+      if (UpdateCurrentCharacterIfPossiblePrefixMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(UpdateCurrentCharacterIfPossiblePrefixMethodInfo));
+        return false;
+      }
+        
+      if (UpdateCurrentCharacterIfPossiblePostfixMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(UpdateCurrentCharacterIfPossiblePostfixMethodInfo));
+        return false;
+      } 
+        
+      if (InitializeInventoryPostfixMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(InitializeInventoryPostfixMethodInfo));
+        return false;
+      }
+      
+      if (AfterTransferPostfixMethodInfo == null) {
+        PrintMethodNotFoundByReflectionError(nameof(InitializeInventoryPostfixMethodInfo));
+        return false;
+      }
+
+      if (!IsTargetPatchable(UpdateCurrentCharacterIfPossibleMethodInfo, UpdateCurrentCharacterIfPossibleHashes)) {
+        return false;
+      }
+      
+      if (!IsTargetPatchable(InitializeInventoryMethodInfo, InitializeInventoryHashes)) {
+        return false;
+      }
+      
+      if (!IsTargetPatchable(AfterTransferMethodInfo, AfterTransferHashes)) {
+        return false;
+      }
+
+      return base.IsApplicable(game);
+    }
+
+    private static void MapLeftAndRightInventory(SPInventoryVM spInventoryVm, Action<SPItemVM> itemMapper) {
+      var rightItemListVm = (MBBindingList<SPItemVM>) RightItemListVmFieldInfo.GetValue(spInventoryVm);
+      var leftItemListVm = (MBBindingList<SPItemVM>) LeftItemListVmFieldInfo.GetValue(spInventoryVm);
```

Comment:
Use `.BuildGetter<SPInventoryVM,MBBindingList<SPItemVM>>()`  for these two accesses, move to static readonly field.